test/extension/handlers/topologymutation/handler.go

```diff
@@ -61,8 +64,8 @@ func NewExtensionHandlers(scheme *runtime.Scheme) *ExtensionHandlers {
 		decoder: serializer.NewCodecFactory(scheme).UniversalDecoder(
 			infrav1.GroupVersion,
 			vmwarev1.GroupVersion,
-			controlplanev1.GroupVersion,
-			bootstrapv1.GroupVersion,
+			controlplanev1beta1.GroupVersion,
+			bootstrapv1beta1.GroupVersion,
```

Comment:
[Not sure] I assume the goal is to move the runtime extension to v1beta2

Let's also use v1beta2 here? (apparently it doesn't really matter, but v1beta1 doesn't seem to make sense?)